README.md

```diff
@@ -93,10 +93,10 @@ You need to deploy the functions associated with the Callback and Voicemail Flex
 
 #### Pre-deployment Steps
 
-1. From the root directory of your copy of the source code, change into `public/resources` and rename `.env.example` to `.env`.
+1. From the root directory of your copy of the source code, change into `serverless` and rename `.env.example` to `.env`.
 
 ```
-cd public/resources && mv .env.example .env
+cd serverless && mv .env.example .env
```

Comment:
nitpick: `mv serverless/.env.example serverless/.env` no need for `cd` first and easier to run and keeps you in the main folder

Reply:
Hey @ktalebian I think @bermudezmt designed it this way because she wants people to `cd` into that folder in order to execute the deploy command at step 3. If we remove the `cd` from there, we need to add it back before step 3, so we might as well keep it here. Does that make sense?

Reply:
ah I see. Thanks for the that. Okay +1. Merge away!